snapcraft.yaml

```diff
@@ -0,0 +1,19 @@
+name: tqdm
+version: '1.0'
```

Comment:
should this be `tqdm`'s version, or is this a `snap` build number?

Reply:
You can use any version you like - it carries no semantic meaning in snaps. You can make this snap revision or match to your application.